RealmModelGenerator/EntitiesView.swift

```diff
@@ -8,24 +8,45 @@
 
 import Cocoa
 
+@objc protocol EntitiesViewDataSource: NSObjectProtocol {
+    optional func numberOfRowsInEntitiesView(entitiesView:EntitiesView) -> Int
+    optional func entitiesView(entitiesView:EntitiesView, titleForEntityAtIndex index:Int) -> String?
+}
+
+@objc protocol EntitiesViewDelegate: NSObjectProtocol {
+    optional func addEntityInEntitiesView(entitiesView:EntitiesView)
+    optional func removeEntityInEntitiesView(entitiesView:EntitiesView, index:Int)
+    optional func entitiesView(entitiesView:EntitiesView, didRemoveEntityAtIndex index:Int)
+}
+
 @IBDesignable
 class EntitiesView: NibDesignableView, NSTableViewDelegate, NSTableViewDataSource {
     
     @IBOutlet var tableView:NSTableView!
-    
     @IBOutlet var addButton:NSButton!
     @IBOutlet var removeButton:NSButton!
     
+    weak var dataSource:EntitiesViewDataSource?
+    weak var delegate:EntitiesViewDelegate?
+    
+    var selectedIndex:Int?
```

Comment:
can we make this a computed property that calls self.tableView.selectedIndex instead of manually keeping track of it.